core/compatible.py

```diff
@@ -71,15 +71,15 @@ def check_for_requirements(start_api_server):
     api_config = api_configuration()
     connection_timeout = api_config["api_database_connection_timeout"]
     try:
-        import pymongo
-        import netaddr
-        import flask
-        del netaddr
-        del flask
+        with open(os.path.join(os.getcwd(), 'requirements.txt'), 'r') as requirements:
+            for dependency in requirements:
+                dependency_name = dependency.split('=')[0]
+                __import__(dependency_name)
     except Exception:
         exit_failure("pip install -r requirements.txt")
     # check mongodb
     try:
+        import pymongo
```

Comment:
one import for a file should be fine right? i guess pymongo is already imported in the upper try